src/services/searchWatches.ts

```diff
@@ -0,0 +1,11 @@
+import { AxiosInstance } from 'axios'
+import {Watch} from './types/Watch';
+
+
+export const SearchWatches = async (api: AxiosInstance, req = ''): Promise<Watch[] | undefined> => {
```

Comment:
Should be *listWatchesWithQuery* or *findWatches* or *searchWatches*, but keep the first letter lowercase.